packages/constants/src/mcf.ts

```diff
@@ -298,3 +298,25 @@ export const COMPANY_ADDRESS_PURPOSES: ICompanyAddressPurpose[] = [
   {id: COMPANY_ADDRESS_PURPOSE.OPERATING, purpose: 'operating'},
   {id: COMPANY_ADDRESS_PURPOSE.CORRESPONDENCE, purpose: 'correspondence'},
 ];
+
+/*************************************
+ * Job Application Statuses
+ *************************************/
+
+export enum APPLICATION_STATUS {
+  NOT_SENT = 0,
+  UNDER_REVIEW,
+  SUCCESSFUL,
+  UNSUCCESSFUL,
+  RECEIVED,
+  WITHDRAWN,
+}
+
+export const JOB_APPLICATION_STATUS = {
```

Comment:
And this one to JOB_APPLICATION_STATUSES

Reply:
updated thanks!